src/api/articles.ts

```diff
@@ -8,6 +8,7 @@ export interface Tags {
 export interface Tag {
   name: string
   image: string
+  blurhash: string
 }
```

Comment:
penso si possano levare dato che le due interfacce inerenti ai tags sono in tags.ts

Reply:
sì questo fixato con il merge dell'hook useApiCall, il file tags.ts è stato direttamente cestinato lol